File: CS/PersonalOrganizer/ContactContextDataModel/Model/IUnitOfWorkFactory.cs
using System;
using System.Linq;
using System.Data;
using System.Data.Entity;
using System.Linq.Expressions;
using System.Collections.Generic;
using PersonalOrganizer.Common.Utils;
using PersonalOrganizer.Common.DataModel;
using PersonalOrganizer.Common.DataModel.EntityFramework;
using PersonalOrganizer.Model;

namespace PersonalOrganizer.ContactContextDataModel {
    public interface IUnitOfWorkFactory : IUnitOfWorkFactory<IContactContextUnitOfWork> {
    }
}

File: VB/PersonalOrganizer/ContactContextDataModel/Model/IUnitOfWorkFactory.vb
Imports System
Imports System.Linq
Imports System.Data
Imports System.Data.Entity
Imports System.Linq.Expressions
Imports System.Collections.Generic
Imports Common.Utils
Imports Common.DataModel
Imports Common.DataModel.EntityFramework
Imports PersonalOrganizer.Model
Namespace ContactContextDataModel
    Public Interface IUnitOfWorkFactory
        Inherits IUnitOfWorkFactory(Of IContactContextUnitOfWork)
    End Interface
End Namespace
